packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -610,9 +610,10 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
     if (_snackBars.isEmpty || _snackBarController.status == AnimationStatus.dismissed)
```

Comment:
docs for this method should be updated to talk about the issue of the timing too